public/pages/MonitorDetails/containers/MonitorHistory/MonitorHistory.js

```diff
@@ -243,10 +242,10 @@ class MonitorHistory extends PureComponent {
         monitorIds: monitorId,
       };
 
-      const resp = await httpClient.get(`../api/alerting/alerts?${queryString.stringify(params)}`);
+      const resp = await httpClient.get('../api/alerting/alerts', { query: params });
```

Comment:
Do we need to `stringify` this?

Reply:
This is a little wired, in the HTTP request there can be `params`, `query`, and `body`, 3 types of value. Only the value of `body` needs to be formatted to JSON string. I didn't take much look into the source code, but just tried several times and find that.

Reply:
Ok that is good to know, thanks